am/Chart.yaml

```diff
@@ -24,3 +24,5 @@ annotations:
   # https://artifacthub.io/packages/helm/graviteeio/am?modal=changelog
   artifacthub.io/changes: |
     - Add upgrader framework job
```

Comment:
Does this feat (upgrader fwk), added in 1.0.51 ? thinking

Reply:
This was already on master